apps/web/rollup.config.workers.mjs

```diff
@@ -1,20 +1,41 @@
+import { defineConfig } from 'rollup'
+
+// Common Plugins
 import { nodeResolve } from '@rollup/plugin-node-resolve'
 import typescript from 'rollup-plugin-typescript2'
-import { defineConfig } from 'rollup'
+import commonjs from '@rollup/plugin-commonjs'
+import json from '@rollup/plugin-json'
+import manifestPlugin from 'rollup-plugin-output-manifest'
+const outputManifest = manifestPlugin.default
+
+console.log("outputManifest", outputManifest)
+
+// Production plugins
+import terser from '@rollup/plugin-terser'
+
+const isProduction = process.env.NODE_ENV === 'production'
 
 export default defineConfig({
+  name: 'ReadMetadataWorker',
   input: 'src/workers/readMetadata.ts',
   output: {
+    entryFileNames: 'readMetadata.[hash].js',
```

Comment:
Hashed metadata. Now it can be served as an independent script from nextjs public folder. No messing with nextjs build system at all